packages/wrangler/src/__tests__/r2.test.ts

```diff
@@ -278,10 +340,8 @@ describe("r2", () => {
 					`r2 object put bucketName-object-test/wormhole-img.png --file ./wormhole-img.png`
 				)
 			).rejects.toThrowErrorMatchingInlineSnapshot(`
-			"Error: Wrangler only supports uploading files up to ${prettyBytes(
-				MAX_UPLOAD_SIZE
-			)} in size
-			wormhole-img.png is ${prettyBytes(TOO_BIG_FILE_SIZE)} in size"
+			"Error: Wrangler only supports uploading files up to 315 MB in size
+			wormhole-img.png is 316 MB in size"
```

Comment:
Somehow this snapshot was out of date, so I updated it

Reply:
To update it, did you run `npm run test -- -u`?

Reply:
Yep!